features/unit/v2indexerclient_paths.feature

```diff
@@ -5,15 +5,14 @@ Feature: Indexer Client v2 Paths
 
   @unit.indexer
   Scenario Outline: LookupAssetBalances path
-    When we make a Lookup Asset Balances call against asset index <index> with limit <limit> afterAddress "<afterAddress>" round <round> currencyGreaterThan <currencyGreaterThan> currencyLessThan <currencyLessThan>
```

Comment:
why was round removed?

Reply:
this parameter was removed from indexer since it was never implemented

Reply:
This would break an existing test, make sure to update it instead of generating a new step

Reply:
Update which test?

Reply:
If you run them locally, your test fails. If you cannot see this, then either the sdk is not implementing this test, or you will see it fail once this job gets merge.

Reply:
Search for "we make a Lookup Asset Balances call against asset index" in the sdk, and you'll find the test.

Reply:
The test in the sdk, got it. I changed it in go sdk.